docs/riff_streaming_inmemory-gateway_create.md

```diff
@@ -4,11 +4,11 @@ title: "riff streaming inmemory-gateway create"
 ---
 ## riff streaming inmemory-gateway create
 
-create a in-memory gateway of messages
+create an in-memory gateway of messages
 
 ### Synopsis
 
-<todo>
+Creates an in-memory Gateway within a namespace.
```

Comment:
Is "within a namespace" present in similar commands? Seems implied to me. I'm fine with either, as long as it is consistently used/not-used.

Reply:
Yes it exists in other commands. I think it's implied as well, but was asked to add is in previous reviews.